docs/developer/decision-records/2023-05-25-template-repository/README.md

```diff
@@ -0,0 +1,90 @@
+# Template and `.github` Repositories
+
+## Decision
+
+Two new repositories will be created within the EDC GitHub organization:
+- `.github` repository for common and universal documents,
+- dedicated [template repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) for mandatory (repo-specific) files.
+
+## Rationale
+
+Currently, generally valid documents like the `pr_etiquette.md` or `CONTRIBUTING.md`
+are located at the Connector repository, although they apply to all repositories in the scope of the
+EDC project. In addition, resources like style files or issue/pr templates are duplicated across
+repositories and, with this, are not up-to-date, as most of the changes to those documents are made
+in the Connector repo and are not transferred accordingly.
+
+The reasoning for moving these documents out to separate repositories is as follows:
+- simplify maintenance of files
+- facilitate the entrance for new community members
+- improve automation of GitHub processes
+- solidify creation process of new EDC repositories
+
+## Approach
+
+At the time of this writing, two repositories can be identified:
+- `.github`
+- `template-basic`
+
+We need to identify "common" and "repository-specific" documents. In addition, we need to test how the
+`.github` repo and the repositories' `.github` folders relate to each other.
+
+### Common documents
+
+Suggested structure for `.github`:
+- `.github/`:
+    - [ISSUE_TEMPLATE](https://github.com/eclipse-edc/Connector/tree/main/.github/ISSUE_TEMPLATE) and [PULL_REQUEST_TEMPLATE.md](https://github.com/eclipse-edc/Connector/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+    - generic workflows, e.g.
```

Comment:
maybe we could also include the `release-<COMPONENT>.yml` workflow?

Reply:
This was not meant to be a finite list...